pennylane/templates/subroutines/out_adder.py

```diff
@@ -73,6 +82,33 @@ def circuit():
 
     The result :math:`[1 0 0]`, is the ket representation of
     :math:`5 + 6 \, \text{modulo} \, 7 = 4`.
+
+    .. details::
+        :title: Usage Details
+
+        This template takes as input four different sets of wires.
+
+        The first one is ``x_wires`` which is used
+        to encode the integer :math:`x < mod` in the computational basis. Therefore, we need at least
+        :math:`\lceil \log_2(x)\rceil` ``x_wires`` to represent :math:`x`.
+
+        The second one is ``y_wires`` which is used
+        to encode the integer :math:`y < mod` in the computational basis. Therefore, we need at least
+        :math:`\lceil \log_2(y)\rceil` ``y_wires`` to represent :math:`y`.
+
+        The third one is ``output_wires`` which is used
+        to encode the integer :math:`b+x+y (mod)` in the computational basis. Therefore, we need at least
```

Comment:
y and mod are too close here:

<img width="144" alt="Captura de pantalla 2024-08-28 a las 15 59 38" src="https://github.com/user-attachments/assets/34e986fb-c8a5-4a4c-8b90-ece7295c93ee">

You can do that with `\quad` or `\hspace{}`